packages/main/src/RangeSlider.js

```diff
@@ -112,6 +114,8 @@ const metadata = {
  * @author SAP SE
  * @alias sap.ui.webcomponents.main.RangeSlider
  * @extends SliderBase
+ * @formAssociated
+ * @formProperties startValue endValue
```

Comment:
fyi @g-cheishvili RangeSlider has two form associated properties, all the rest have a single one ("value" or "checked").